static/src/javascripts/bootstraps/standard/main.js

```diff
@@ -118,6 +118,7 @@ define([
             if (guardian.isEnhanced) {
                 ab.segmentUser();
                 ab.run();
+                ab.trackEvent();
```

Comment:
I don't think so - segmenting users, running tests, and firing the tracking event are distinct operations.
Would there be any benefit in conflating it with `run`?